modules/lti-service-impl/src/main/java/org/opencastproject/lti/service/impl/LtiServiceImpl.java

```diff
@@ -493,6 +493,12 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
       } else {
         this.retractWorkflowId = retractWorkflowId;
       }
+      final String copyWorkflowId = (String) properties.get("copy-workflow-id");
+      if (copyWorkflowId == null) {
+        this.copyWorkflowId = "copy-event-to-series";
+      } else {
+        this.copyWorkflowId = copyWorkflowId;
+      }
```

Comment:
Java's [`Objects.toString(Object o, String nullDefault`](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#toString-java.lang.Object-java.lang.String-) is a very convenient way of doing exactly what you did:

```suggestion
      this.copyWorkflowId = Objects.toString(properties.get("copy-workflow-id"), "copy-event-to-series");
```

Reply:
Thanks! I changed that.